docs/components/connectors/introduction.md

```diff
@@ -2,125 +2,45 @@
 id: introduction-to-connectors
 title: Introduction
 description: "A Connector is a reusable building block that performs the integration with an external system and works out of the box."
+keywords: [bridge, reusable component, low code]
 ---
 
-A **Connector** is a reusable building block that performs the integration with an external system and works out of the box.
+A **Connector** is a reusable building block that performs the integration with an external system.
```

Comment:
I'm still not happy with this super brief, crisp definition. Once we have one we like, we'll need to update it here and the getting started guide - https://docs.camunda.io/docs/guides/orchestrate-api-endpoints/

Reply:
A few thoughts...
- "A Connector is an integrative feature..."
- "A supporting Component feature"
- "A supplemental Component feature"
- "An integrant company feature" (though integrant may not be a well-known vocabulary term)
- Would we at all consider "sub-Component" or "child feature" is this not quite right?
- "Interaction mechanism"
- I've seen "[facilities component](https://www.computerweekly.com/blog/Open-Source-Insider/What-is-a-software-connector)" used, but not frequently.